definitions/procedures/backup/pulp.rb

```diff
@@ -26,11 +29,34 @@ def run
 
     private
 
+    def current_pulp_feature
+      if feature(:pulp2) && feature(:pulpcore)
+        return choose_pulp2_pulp3
+      end
+
+      feature(:pulp2) || feature(:pulpcore)
+    end
+
+    def choose_pulp2_pulp3
```

Comment:
@jlsherrill Like to know if this makes sense? if system will be in state where user will need to decide which pulp version to backup?

Reply:
I believe the user is advised to make a backup just before the migration. I have some additional questions: When is the backup of content migrated to pulp3 expected to be used for restore, after upgrade? What is the benefit? (Maybe a backup of a very long and successful migration?) What about backups taken during the migration?

Reply:
> @jlsherrill Like to know if this makes sense? if system will be in state where user will need to decide which pulp version to backup?

I have removed this choose_pulp2_pulp3 method as per latest changes in pr. As per the latest changes I want to achieve,
1. If 6.9 in freshly installed we should not take backup of pulpcore as that wont help and does not make sense.
2. Prior to migration of pulpcore we should have backup of pulp2.
3. After migration to pulpcore on 6.9, we would want user to go to 6.10 with backup. If user takes backup at this time, it should be of pulpcore.

Request to review, thank you!